ui/raidboss/data/05-shb/trial/castrum-ex.js

```diff
@@ -0,0 +1,123 @@
+
+export default {
+  zoneId: ZoneId.CastrumMarinumExtreme,
+  triggers: [
+    {
+      id: 'CastrumEx Optimized Ultima',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5B10', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'CastrumEx Aetheroplasm Production',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55AA', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Get orbs',
+          cn: '撞球',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Divide Et Impera P1',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '555B', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'CastrumEx Divide Et Impera P2',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5537', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '八方位',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Primus Terminus Est',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C3', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '靠边',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Tertius Terminus est',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55CC', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '注意落剑顺序',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Sidescathe Left',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55D5', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '左(看翅膀)',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Full-Power Optimized Ultima',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5B17', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '凉凉',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Sidescathe Right',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55D4', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '右（看翅膀）',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Secundus Terminus est',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C8', capture: false }),
+      alarmText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '靠边放剑(十字四角)',
```

Comment:
Are there [headmarker](https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#1b-networktargeticon-head-markers) lines for this mechanic? If so, then you could tell the X players to go to cardinals and + to go intercardinals.

Reply:
Yes, there are, but I'm not sure which one ties to which as I'm in the office and using a mac so I couldn't fire up ACT...
I'm not that familiar with network codes for ffxiv
55CA and 55C9 are the corresponding markers ability id

here are the logs for them:
[00:12:40.630] 15:400072BD:Bitblade:55CA:Secundus Terminus Est:1067C518:Shy Non:710003:43000400:1B:55CA8000:0:0:0:0:0:0:0:0:0:0:0:0:124402:124402:10000:10000:0:1000:83.11797:95.16481:-0.01531982:0.9456657:148000:148000:10000:10000:0:1000:82.6748:95.90427:0:-4.792213E-05:00006E95
[00:12:40.630] 15:400072BE:Bitblade:55C9:Secundus Terminus Est:106C8B00:Mina Faewalker:710003:4DCE0400:1B:55C98000:0:0:0:0:0:0:0:0:0:0:0:0:177616:177616:3400:10000:0:1000:100.9689:82.07825:0:3.056743:148000:148000:10000:10000:0:1000:100.973:82.10254:0:-4.792213E-05:00006E96